src/TodoListItem.js

```diff
@@ -2,9 +2,10 @@ import React from "react";
 
 
 function TodoListItem(props) {
-  console.log('todolistitem is rendered');
+  //Destructure props 
+  const { todo } = props;
```

Comment:
This is working well! 
As we have just one(or couple) props, we can do it destructure at the top level. Like this:
`function TodoListItem({todo}){`
And same for other components with props